### ISSUE TITLE
Sly 3 cutscenes cutting audio

### ISSUE THREAD

**Issue by AndrewNDR** [RPCS3/rpcs3#12915]
### Quick summary

Every non-rendered cutscene cuts audio off, making it hard to listen to the story

### Attach a log file

[RPCS3.log](https://github.com/RPCS3/rpcs3/files/9922544/RPCS3.log)

### System configuration

CPU: i5-9600k
GPU: GTX 1660
16Gb RAM

**Comment by YuriNator557** [comment]
Please use the discord server or forums when asking for help, you have multiple non-default settings that could be causing this

**Comment by AndrewNDR** [comment]
Ok. But even with default settings still have the issue

**Comment by YuriNator557** [comment]
Confirmed problem, I can also replicate
Will need to find when this started happening

**Comment by AndrewNDR** [comment]
@AniLeo When watching the fmv cutscenes/videos, the audio cuts out for a couple of seconds. It happens even when booting the game, where the book opens - before the "Press Start" screen.

**Comment by AniLeo** [comment]
The need information tag is for the regression testing, assuming it's a regression as mentioned above

**Comment by SlyCooperReloadCoded** [comment]
After some testing, I've found this only applies to the physical copies.  Digital standalone version is unaffected.  Happens with Sly 2 as well, physical copies only, not digital standalone.  Strangely, Sly 1 does not have this issue on physical or digital copies.  Prints this to the log when it happens:

E SYS: 'cellAudioInit' failed with 0x80310701 : CELL_AUDIO_ERROR_ALREADY_INIT [1]
E SYS: 'cellAudioPortStop' failed with 0x80310708 : CELL_AUDIO_ERROR_PORT_NOT_RUN [1]

This might not be related to this exact issue, but it is an issue about Sly Cooper FMV audio, so I'll mention it just in case it happens to be a part of the same regression - the Japanese physical copy (BCJS30061) has all of Sly 1's FMVs completely silent.

**Comment by Megamouse** [comment]
Is this still an issue?

**Comment by SlyCooperReloadCoded** [comment]
The issue still exists, but to a lesser degree.  It's still limited to the physical releases, and only Sly 2 and 3 on the Japanese copy (all three on the American copy) are affected, but after the errors appear and the audio cuts out it manages to recover, but this is only true for the two legal/brand cutscenes at the start.  On at least American and Japanese Sly 3, the rest of the cutscenes play fine.

**Comment by Jonathan44062** [comment]
How can you know this only affects the Disc version and not the PSN version? Is there a difference between them? As if the digital version is updated while the disc version is not?

**Comment by SlyCooperReloadCoded** [comment]
Yes, digital version is actually different and has a feature physical doesn't - Overscan.  It also has some unique audio issues of its own, but on a software level, not an emulator level.

**Comment by digant73** [comment]
@AndrewNDR is that still an issue?

**Comment by Jonathan44062** [comment]
Completed 2 episodes so far, no audio issues anymore, it seems this odd problem it got fixed, we can close this
